### PR TITLE
Support package elements and sorting

### PR DESCRIPTION
**Pull request by nolanpro**
See https://processmaker.atlassian.net/browse/FOUR-13174

Required PRs
| CI Tag | PR | Status |
|--------|----|-------|
| ci:processmaker:feature/FOUR-13174 | https://github.com/ProcessMaker/processmaker/pull/5978 | :white_check_mark: |
| ci:package-signature:feature/FOUR-13174 | https://github.com/ProcessMaker/package-signature/pull/36 | :white_check_mark: |
| ci:package-ai:feature/FOUR-13174 | https://github.com/ProcessMaker/package-ai/pull/68 | :white_check_mark: |
| ci:package-files:feature/FOUR-13174 | https://github.com/ProcessMaker/package-files/pull/117 | :white_check_mark: |
| ci:package-photo-video:feature/FOUR-13174 | https://github.com/ProcessMaker/package-photo-video/pull/27 | :white_check_mark: |
| ci:package-webentry:feature/FOUR-13174 | https://github.com/ProcessMaker/package-webentry/pull/222 | :white_check_mark: |
| ci:package-googleplaces:feature/FOUR-13174 | https://github.com/ProcessMaker/package-googleplaces/pull/44 | :white_check_mark: |
| ci:package-savedsearch:feature/FOUR-13174 | https://github.com/ProcessMaker/package-savedsearch/pull/399 | :white_check_mark: |
| ci:package-plaid:feature/FOUR-13174 | https://github.com/ProcessMaker/package-plaid/pull/8 | :white_check_mark: |

ci:next
ci:deploy
.........

### CODE REVIEW

File: src/components/vue-form-builder.vue [hunk 1]
```diff
@@ -762,31 +644,48 @@ export default {
...
-        if (indexA !== -1) {
-          return -1;
+      return filtered;
+    },
+    filteredControlsGrouped() {
+      const grouped = this.filteredControls.reduce((groups, control) => {
+        let groupName = _.get(control, 'group', null);
+        
+        if (!groupName) {
+          groupName = "Advanced";
```
Comment (caleeli) [inline src/components/vue-form-builder.vue:650]: Since "Advanced" is an special group and it is used in different places, consider to move to a constant

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-100a38a30b.engk8s.processmaker.net

**Review by caleeli — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-100a38a30b.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-100a38a30b.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-100a38a30b.engk8s.processmaker.net

**Pull request merged**